sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/_simulator.py

```diff
@@ -24,8 +24,8 @@
 from ._helpers import ConversationHistory, Turn
 from ._utils import JsonLineChatProtocol
 
-if len(USER_AGENT) < 24:
-    USER_AGENT += "/simulator"
+
+USER_AGENT += " (type=simulator)"
```

Comment:
Isn't this changing the global USER_AGENT? Would this cause problem for evaluators?

Reply:
I don't think so.

I added print statements:
- PromptyEvaluatorBase line 54 after USER_AGENT is used
```python
print("************")
print(USER_AGENT)
print("************")
```
And in the simulator where I'm making the change        


Here's a little bit of the outputs:
```
************
azure-ai-evaluation/1.1.0 (type=simulator)
************
Class Simulator: This is an experimental class, and may change at any time. Please see https://aka.ms/azuremlexperimental for more information.
/Users/nagkumar/Documents/msft.nosync/azure-sdk-for-python/sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/_simulator.py:153: UserWarning: You have specified 'num_queries' > len('tasks') (4 > 3). All tasks will be used for generation and the remaining 1 lines will be simulated in task-free mode
  warnings.warn(
Generating: 100%|████████████████████████████████████████████████| 3/3 [00:18<00:00,  6.22s/message]
************
azure-ai-evaluation/1.1.0
************
************
azure-ai-evaluation/1.1.0
************
```

The USER_AGENT in the _base_prompty_eval.py is being imported like this:
`from ..._user_agent import USER_AGENT`

Hence changing the value for `USER_AGENT` in the _simulator.py will not affect the value of `USER_AGENT` in any other imports.

Reply:
Thanks for checking on this. Appreicate it!